api/kotlinx.reflect.lite.api

```diff
@@ -0,0 +1,226 @@
+public abstract interface class kotlinx/reflect/lite/KAnnotatedElement {
+}
+
```

Comment:
Did a shallow review of API, other than one data class nothing caught my eye. Haven't checked for anything deep though.